src/main/scala/gemmini/GemminiISA.scala

```diff
@@ -17,4 +17,21 @@ object GemminiISA {
   val CONFIG_LOAD = 1.U
   val CONFIG_STORE = 2.U
   val CONFIG_EX = 0.U
+
+  //==========================================================================
+  // cisc-gemmini opcodes
+  //==========================================================================
+  val CISC_CONFIG  = 10.U(7.W) // same as COMPUTE_AND_FLIP
+  val ADDR_AB      = 11.U(7.W)
+  val ADDR_CD      = 12.U(7.W)
+  val SIZE0        = 13.U(7.W)
+  val SIZE1        = 14.U(7.W)
```

Comment:
What do 0 and 1 stand for here?

Reply:
SIZE0 command sets the M and N dimensions and SIZE1 command sets the K dimension. 

M,N,K being 32-bit integers, so they could be merged into a single SIZE command

Reply:
Do you want to name them `SIZE_MN` and `SIZE_K` in that case?

Reply:
yes